dev/build/windows/MakeCoq_MinGW.bat

```diff
@@ -40,7 +40,7 @@ REM see -destcyg in ReadMe.txt
 SET DESTCYG=C:\bin\cygwin_coq
 
 REM see -destcoq in ReadMe.txt
-SET DESTCOQ=C:\bin\coq
+SET DESTCOQ=C:\bin\the coq proof assistant

```

Comment:
Are you sure about this? It looks like something that goes beyond strengthening CI.

Reply:
Yup we should make this conditional to running in PRs for example.

I anticipate having this PR working will be lots of "fun" anyways.

Reply:
Afaik the default path is not used. On the INRIA runners we are using random paths.
The defaults are more intended to be convenient for users.
If the CI path needs to be changed, it needs to be done in the CI file.

One more note: this test will not be terribly effective because this path is only used for installation. During build a relative path is used. This is because a Coq which can be installed into random folders needs to have relative paths inside. The only place where this path is used is during make install.

If we really want to test this, we actually need to install Coq somewhere and this path should have spaces in it.